freqtrade/persistence/migrations.py

```diff
@@ -294,6 +295,28 @@ def fix_old_dry_orders(engine):
         ).values(ft_is_open=False)
         connection.execute(stmt)
 
+        # Update current Order where
+        # -current Order is open
+        # -current Order trade_id not equal to current Trade.id
+        # -current Order not stoploss
+        # -order_id not equal to current Trade.stoploss_order_id
+        # -current Order is dry
+
+        # NEW WIP
+        stmt = update(Order).where(
```

Comment:
What's the reason for this update? (i understand what it does, but it seems unnecessary unless i miss something?)

dry-run orders can stay open without problems - and will be loaded from the database.

Reply:
I was trying to replicate existing code but without "open_order_id" and the new logic. You mean I can leave migration functions unchanged and it will be remove after?

Reply:
well i'd leave it for now and work on regular functionality (backtesting, trading).

There's WAY more important thing to do than THIS small update (which is per se not REALLY necessary unless you migrate from a VERY old database).